infrastructure/cli/initializer.go

```diff
@@ -66,7 +66,7 @@ func (i *Initializer) Init() error {
 	logger := log.With().Str("method", "cli.Init").Logger()
 	cliInstalled := config.CurrentConfig().CliSettings().Installed()
 	logger.Debug().Str("cliPath", cliPathInConfig()).Msgf("CLI installed: %v", cliInstalled)
-	if !config.CurrentConfig().ManageBinariesAutomatically() {
+	if !config.CurrentConfig().ManageCliBinariesAutomatically() {
```

Comment:
This is very nitpicky of me. But it is only one binary. So it should most likely be `ManageCliBinaryAutomatically`. Not a showstopper, though.